pyproject.toml

```diff
@@ -103,6 +103,7 @@ isort = ">=5.10.1"
 pytest = ">=7.1.2"
 pytest-cov = ">=4.1.0"
 pytest-split = ">=0.8.0"
+polars = ">=1.31.0"
```

Comment:
For now, just placing in the dev dependencies for others to play around with - will move to main dependencies if we're fine with this